istresearch/rapidpro
# P4-1511 - one off, adding postmaster svg

# For the Reviewer

When this is complete, you should approve the PR via github.

# For the Reviewee

https://istresearch.atlassian.net/browse/P4-1511
Adding SVG images for postmaster glyph icons

## Summary
Adding SVG images for postmaster glyph icons

#### Release Note
<Concise sentence describing change>Required.

#### Breaking Changes
<Description for Techops of how to handle changes, migrations, updates>None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification
1) Standup an engage image running this branch.
2) Navigate to the *{subdir}*/channels/types/postmaster/claim path
3) Verify that the postmaster link includes an icon as seen below:
<img width="1166" alt="Screen Shot 2020-05-01 at 11 10 13 AM" src="https://user-images.githubusercontent.com/6886738/80816069-d735ae80-8b9c-11ea-9cd1-06b29e2c9ac3.png">

4) Click on the postmaster link and verify that the claim form includes the icon:
<img width="571" alt="Screen Shot 2020-05-01 at 11 10 22 AM" src="https://user-images.githubusercontent.com/6886738/80816086-db61cc00-8b9c-11ea-98eb-b528fdd7b772.png">

The 4.0.2 release will eventually look like this:
<img width="1141" alt="Screen Shot 2020-05-01 at 11 10 30 AM" src="https://user-images.githubusercontent.com/6886738/80816096-df8de980-8b9c-11ea-8a42-40e146823e75.png">
